harvard-lts/drs-import-management-service
# Integration test with Dataverse

**Integration test with Dataverse**
* * *

**GitHub Issue**: ([link](https://app.zenhub.com/workspaces/hdc-62163f60fb08c60011fa4795/issues/harvard-lts/hdc/167))

# What does this Pull Request do?
Adds pending integration test with Dataverse, which covers the Dataverse status report operation.

# How should this be tested?

A description of what steps someone could take to:
* Make sure test .env variables are updated based on .env.example files
* Rebuild test environment: docker-compose -f docker-compose-test.yml build --no-cache
* Run the local test environment: `docker-compose -f docker-compose-test.yml up`
* Execute the integration test: `docker exec -it test_runner pytest test/integration/ingest/infrastructure/api/test_dataverse_ingest_status_api_client.py`

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests N/A
- integration tests Yes
- functional tests N/A

# Interested parties
@jessjass @ives1227 @adaybujeda

keyword: resolved
